feature/migration/qrcode/src/main/kotlin/app/k9mail/feature/migration/qrcode/AccountData.kt

```diff
@@ -0,0 +1,105 @@
+package app.k9mail.feature.migration.qrcode
+
+import app.k9mail.core.common.mail.EmailAddress
+import app.k9mail.core.common.net.Hostname
+import app.k9mail.core.common.net.Port
+
+internal data class AccountData(
```

Comment:
I was wondering why this looks similarly to `QrCodeData` but differs slightly and found that the validation step also transforms `QrCodeData` into `AccountData` which is surprising for a validator. I think this should be 2 distinct responsibilities.